addons/html_editor/static/src/editor/core/history_plugin.js

```diff
@@ -2,37 +2,580 @@
 
 import { registry } from "@web/core/registry";
 import { Plugin } from "../plugin";
+import { setSelection } from "../utils/selection";
+import { serializeSelection } from "../utils/serialize";
 
 export class HistoryPlugin extends Plugin {
     static name = "history";
-    static shared = ["enableObserver", "disableObserver"];
+    static shared = ["enableObserver", "disableObserver", "makeStep"];
```

Comment:
Do you need to share enableObserver and enableObserver ? You don t have any use case

Reply:
forgot to removed them, they are gone now